recipe/meta.yaml

```
@@ -59,9 +59,8 @@ requirements:
     - xattr >=1.0.0,<2.0.0    # [osx]
 
     # Keep the package noarch
-    - __linux  # [linux]
-    - __osx    # [osx]
-    - __win    # [win]
+    - __unix  # [unix]
+    - __win   # [win]
```

IOW keeping the Linux and macOS distinction here

```suggestion
    - __linux  # [linux]
    - __osx    # [osx]
    - __win    # [win]
```